doc/sphinx/intro-wrapper.html.rst

```diff
@@ -0,0 +1,6 @@
+
+------------
+Introduction
+------------
+
+.. include:: introduction.rst
```

Comment:
Quick question: why do you need this wrapper file?
Also, it would probably be better to rename this wrapper to introduction.rst and to include a file called `intro-contents.rst` instead.  This way, existing links won't break and the URL won't say `intro-wrapper.html`

Reply:
The HTML and Latex versions require different formats for the title.  HTML:

```
------------
Introduction
------------
```

and Latex:

```
Introduction
------------
```

> Also, it would probably be better to rename this wrapper to introduction.rst

Will do.